packages/codegen/src/codegen/utils.ts

```diff
@@ -13,7 +13,7 @@ export async function loadDocumentModel(
       documentModel = file.state.global;
     } else if (path.endsWith(".json")) {
       const data = fs.readFileSync(path, "utf-8");
-      const document = JSON.parse(data) as DocumentModelState;
```

Comment:
This doesn't break the zip loading?

Reply:
only if the zip file ends with .json, see line 14

Reply:
Nice, there is an eslint error due to data being any.

Reply:
Ah I think this was done to support generating from the spec.json file. I think we need a check that gets the document state on both cases.